snippets/snippets.json

```diff
@@ -543,7 +543,8 @@
             "        LockHL=  OFF",
             "        LockIV=  OFF",
             "        LockLL=  OFF",
-            "UseFactorWeight=  0\r"
+            "UseFactorWeight=  0\r",
+            "        PVPred=  OFF"
         ],
         "description": "For SEPTIC >= v2.82"
```

Comment:
"For SEPTIC >= v2.89" er vel korrekt her. Og så bør det helst være en snippet kalt mvr_v2.88 med description "For SEPTIC <= v2.88" som er lik denne før oppdateringen.